spec/index.bs

```diff
@@ -6335,8 +6335,7 @@ dictionary GPURenderBundleEncoderDescriptor : GPUObjectDescriptorBase {
 interface GPUQueue {
     undefined submit(sequence<GPUCommandBuffer> commandBuffers);
 
-    GPUFence createFence(optional GPUFenceDescriptor descriptor = {});
-    undefined signal(GPUFence fence, GPUFenceValue signalValue);
+    Promise<undefined> onCompletion();
```

Comment:
@jdashg suggested maybe renaming this to something like `onSignalComplete` since this is no longer on a fence object where the context is clear.

I said:

> Alternatively maybe we should try to make this not sound like an event handler (with the word "on" at the front).

E.g. `workCompleted` or something

Reply:
`onWorkCompleted`? I don't know if there's a naming convention for promises in WebIDL.

Reply:
`onPendingWorkCompleted`?

Reply:
As mentioned `on` makes it sound like an event handler. One example (more similar to our device.lost) is:
https://www.w3.org/TR/service-workers/#ref-for-dom-serviceworkercontainer-ready
but other than that most promise-returning methods are verbs, since, you know, they're methods.